components/cards.js

```diff
@@ -258,7 +258,7 @@ const MenuCard = ({ title, image, style, desc, price }) => {
             </View>
             <Image
               source={
-                image === ""
+                image === "" || image == undefined
```

Comment:
I do not recall adding these, but it is good that we are now checking for an undefined image! :D